sharepoint/sharepoint-ps/sharepoint-online/Start-SPOInformationBarriersInsightsReport.md

```diff
@@ -0,0 +1,107 @@
+﻿---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/start-spoinformationbarriersinsightsreport
+applicable: SharePoint Online
+title: Start-SPOInformationBarriersInsightsReport
+schema: 2.0.0
+author: pvrk
+ms.author: pullabhk
+manager: 
+ms.reviewer:
+---
+
+# Start-SPOInformationBarriersInsightsReport
+
+## SYNOPSIS
+
+This cmdlet generates reports in information barriers (IB) meant to identify and discover usage patterns across SharePoint sites and OneDrive accounts in the organization.
+
+## SYNTAX
+
+```powershell
+Start-SPOInformationBarriersInsightsReport [-Yes <Boolean>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet would create reports in Information Barriers (IB) to identify top sites and their modes to help apply suitable controls for the sites as applicable.
+
+## EXAMPLES
+
+### Example 1
+
+```powershell
+PS C:\> Start-SPOInformationBarriersInsightsReport
+```
+
+This PS command would create Information Barriers reports to identify top sites and their modes to help apply suitable controls for the sites as applicable.
+
+## PARAMETERS
+
+### -Yes
```

Comment:
>-Yes [](http://example.com/codeflow?start=4&length=4)

this is an awkwardly named parameter.

so someone uses it like

`-Yes $true`

or 

`-Yes $false`

?